control-plane/pkg/reconciler/source/source.go

```diff
@@ -170,6 +198,18 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, ks *sources.Kafk
 		Spec:       expectedCg.Spec,
 		Status:     cg.Status,
 	}
+
+	// If KEDA is not enabled, then we must update the ConsumerGroup replicas to match KafkaSource consumers
+	// TODO: code below failing unit tests with err: "panic: interface conversion: testing.ActionImpl is not testing.GetAction: missing method GetName"
```

Comment:
not sure I get this todo here, will be addressed later?

Reply:
The commented out code right below isn't working, and panic failing unit tests. But we're using an alternative way right underneath that to check "if KEDA is installed" so adding a temporary TODO for myself to get back to the original check later.